web/src/popover_menus.js

```diff
@@ -441,4 +451,37 @@ export function initialize() {
             message_actions_popover_keyboard_toggle = false;
         },
     });
+
+    // Drafts popover
+    tippy_no_propagation(".drafts-sidebar-menu-icon", {
```

Comment:
Does this popover not have arrow key support?

Reply:
I don't think it does, but once we merge #24622, it should be easy to add.

Reply:
Added!